### PR TITLE
Add Twisted Hollow Encounter Cards

### PR DESCRIPTION
**Pull request by adamgfraser**

### CODE REVIEW

File: pack/fhv/fhvc.json [hunk 1]
```diff
@@ -135,6 +135,448 @@
...
+    "pack_code": "fhvc",
+    "position": 117,
+    "quantity": 1,
+    "shroud": 1,
+    "text": "While an enemy is moving, its location is considered to be connected to Moonlit Clearing.\n<b>Forced<\b> - After you reveal Moonlit Clearing: Test [willpower] (X), where X is the current darkness level. If you fail, find the enemy in pursuit with the highest fight and spawn it at this location.",
+    "traits": "Forest. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The winding paths loop back on themselves.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:435]:
```suggestion
    "back_flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H. P. Lovecraft, \"The Colour Out of Space\"",
```

File: pack/fhv/fhvc.json [hunk 2]
```diff
@@ -135,6 +135,448 @@
...
+    "back_illustrator": "Walter Brocca",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:436]: Please delete this line.

File: pack/fhv/fhvc.json [hunk 3]
```diff
@@ -135,6 +135,448 @@
...
+    "clues": 1,
+    "clues_fixed": true,
+    "code": "10618",
+    "double_sided": true,
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 17,
+    "faction_code": "mythos",
+    "flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
+    "illustrator": "Walter Brocca",
+    "name": "Western Woods",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:447]:
```suggestion
    "name": "Crooked Path",
```

File: pack/fhv/fhvc.json [hunk 4]
```diff
@@ -135,6 +135,448 @@
...
+    "pack_code": "fhvc",
+    "position": 118,
+    "quantity": 3,
+    "shroud": -2,
+    "text": "X is the current darkness level.\n[reaction] After you discover a clue at this location: Look at the revealed side of a connecting location. You may move to that location.",
+    "traits": "Forest. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_illustrator": "Lin Hsiang",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:457]:
```suggestion
    "back_illustrator": "Walter Brocca",
```

Comment (drawntotheflame) [inline pack/fhv/fhvc.json:457]: Please add:

`"back_flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H. P. Lovecraft, \"The Colour Out of Space\"",`

File: pack/fhv/fhvc.json [hunk 5]
```diff
@@ -135,6 +135,448 @@
...
+    "back_name": "Corpse Grove",
+    "back_traits": "Forest. Lair. Dark.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:459]:
```suggestion
    "back_traits": "Forest. Dark.",
```

File: pack/fhv/fhvc.json [hunk 6]
```diff
@@ -135,6 +135,448 @@
...
+    "clues": 1,
+    "code": "10619",
+    "double_sided": true,
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 20,
+    "faction_code": "mythos",
+    "flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:466]: Please delete this line.

File: pack/fhv/fhvc.json [hunk 7]
```diff
@@ -135,6 +135,448 @@
...
+    "illustrator": "Walter Brocca",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:467]:
```suggestion
    "illustrator": "Lin Hsiang",
```

File: pack/fhv/fhvc.json [hunk 8]
```diff
@@ -135,6 +135,448 @@
...
+    "name": "Western Woods",
+    "pack_code": "fhvc",
+    "position": 119,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "<b>Forced</b> - After you reveal Corpse Grove: Take 1 horror.\n<b>Forced</b> - After you successfully investigate Corpse Grove: Take 1 horror.",
+    "traits": "Forest. Dark.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:474]:
```suggestion
    "traits": "Forest. Lair. Dark.",
```

File: pack/fhv/fhvc.json [hunk 9]
```diff
@@ -135,6 +135,448 @@
...
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "Whatever lives here is clearly starved to the point of eating its own kind.",
+    "back_illustrator": "Pixoloid Studios",
+    "back_name": "Bear Den",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:480]:
```suggestion
    "back_name": "Western Woods",
```

File: pack/fhv/fhvc.json [hunk 10]
```diff
@@ -135,6 +135,448 @@
...
+    "position": 120,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "<b>Forced</b> - After you reveal Bear Den: Find the enemy in pursuit with the highest fight and spawn it engaged with you. If you cannot, draw the top card of the encounter deck.",
+    "traits": "Forest. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The knotted old tree has seen far more than you ever will.",
+    "back_illustrator": "Walter Brocca",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:501]: Please delete this line.

File: pack/fhv/fhvc.json [hunk 11]
```diff
@@ -135,6 +135,448 @@
...
+    "back_name": "The Twisted Hollow",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:502]:
```suggestion
    "back_name": "Western Woods",
```

File: pack/fhv/fhvc.json [hunk 12]
```diff
@@ -135,6 +135,448 @@
...
+    "clues": 2,
+    "code": "10621",
+    "double_sided": true,
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 22,
+    "faction_code": "mythos",
+    "flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
+    "illustrator": "Walter Brocca",
+    "name": "Western Woods",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:511]:
```suggestion
    "name": "The Twisted Hollow",
```

File: pack/fhv/fhvc.json [hunk 13]
```diff
@@ -135,6 +135,448 @@
...
+    "code": "10622",
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 23,
+    "faction_code": "mythos",
+    "illustrator": "Igor Kieryluk",
+    "name": "Deep Shadows",
+    "pack_code": "fhvc",
+    "position": 122,
+    "quantity": 2,
+    "test": "<b>Revelation - Test [willpower] (3). This test gets +2 difficulty if your location is [[Dark]]. If you fail, you must either (choose one):\n- Take 2 horror.\n- The investigator who controls the Vale Lantern flips it to its [[Unlit]] side and places it at a connecting location.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:531]:
```suggestion
    "test": "<b>Revelation - Test [willpower] (3). This test gets +2 difficulty if your location is [[Dark]]. If you fail, you must either <i>(choose one)</i>:\n- Take 2 horror.\n- The investigator who controls the Vale Lantern flips it to its [[Unlit]] side and places it at a connecting location.",
```

File: pack/fhv/fhvc.json [hunk 14]
```diff
@@ -135,6 +135,448 @@
...
+    "code": "10624",
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 27,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Stolen Light",
+    "pack_code": "fhvc",
+    "position": 124,
+    "quantity": 1,
+    "text": "Peril.\n<b>Revelation</b> - You must either (choose one):\n- Place 1 doom on the current agenda. This effect can cause the agenda to advance.\n- Flip the Vale Lantern to its [[Unlit]] side. Place the Vale Lantern at the empty location farthest from all investigators.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:559]:
```suggestion
    "text": "Peril.\n<b>Revelation</b> - You must either <i>(choose one)</i>:\n- Place 1 doom on the current agenda. This effect can cause the agenda to advance.\n- Flip the Vale Lantern to its [[Unlit]] side. Place the Vale Lantern at the empty location farthest from all investigators.",
```

**Review by drawntotheflame — COMMENTED** [review]
Good job! Great, very thorough, only a few tweaks. 

The big thing was the inversion of front/back locations - but I've made suggestions to amend that, and hopefully it won't be too tricky for you to work through that. Thank you.

**Comment by adamgfraser** [comment]
@drawntotheflame Updated now!

**Comment by drawntotheflame** [comment]
@Kamalisk this can go live once #1382 is up. Thanks.

File: pack/fhv/fhvc.json [hunk 15]
```diff
@@ -132,9 +132,448 @@
...
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "The silent woods thrum with the sounds of hungry creatures and stalking shadows.",
+    "illustrator": "Rob Laskey",
+    "name": "Deepening Dark",
+    "pack_code": "fhvc",
+    "position": 106,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> - After a location is revealed: Put locations from the top of the Woods deck into play above, below, to the left, and to the right of that location.\n<b>Forced<\b> - When the investigation phase ends: Each investigator at a [Dark]] location must either draw an enemy in pursuit with the highest evade or take 1 horror (cannot be canceled).",
```
Comment (fspoettel) [inline pack/fhv/fhvc.json:154]:
```suggestion
    "text": "<b>Forced</b> - After a location is revealed: Put locations from the top of the Woods deck into play above, below, to the left, and to the right of that location.\n<b>Forced</b> - When the investigation phase ends: Each investigator at a [Dark]] location must either draw an enemy in pursuit with the highest evade or take 1 horror (cannot be canceled).",
```

File: pack/fhv/fhvc.json [hunk 16]
```diff
@@ -132,9 +132,448 @@
...
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 13,
+    "faction_code": "mythos",
+    "illustrator": "Walter Brocca",
+    "name": "Fecund Thicket",
+    "pack_code": "fhvc",
+    "position": 115,
+    "quantity": 1,
+    "shroud": 6,
+    "text": "Fecund Thicket gets -X shroud, where X is the current darkness level.\n<b>Forced<\b> - After you reveal Fecund Thicket: Test [agility] (X), where X is the current darkness level. If you fail, find the enemy in pursuit with the highest printed health and spawn it at this location.",
```
Comment (fspoettel) [inline pack/fhv/fhvc.json:388]:
```suggestion
    "text": "Fecund Thicket gets -X shroud, where X is the current darkness level.\n<b>Forced</b> - After you reveal Fecund Thicket: Test [agility] (X), where X is the current darkness level. If you fail, find the enemy in pursuit with the highest printed health and spawn it at this location.",
```

File: pack/fhv/fhvc.json [hunk 17]
```diff
@@ -132,9 +132,448 @@
...
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 16,
+    "faction_code": "mythos",
+    "illustrator": "Simon Dominic",
+    "name": "Moonlit Clearing",
+    "pack_code": "fhvc",
+    "position": 117,
+    "quantity": 1,
+    "shroud": 1,
+    "text": "While an enemy is moving, its location is considered to be connected to Moonlit Clearing.\n<b>Forced<\b> - After you reveal Moonlit Clearing: Test [willpower] (X), where X is the current darkness level. If you fail, find the enemy in pursuit with the highest fight and spawn it at this location.",
```
Comment (fspoettel) [inline pack/fhv/fhvc.json:429]:
```suggestion
    "text": "While an enemy is moving, its location is considered to be connected to Moonlit Clearing.\n<b>Forced</b> - After you reveal Moonlit Clearing: Test [willpower] (X), where X is the current darkness level. If you fail, find the enemy in pursuit with the highest fight and spawn it at this location.",
```

File: pack/fhv/fhvc.json [hunk 18]
```diff
@@ -132,9 +132,448 @@
...
+    "back_name": "Western Woods",
+    "back_traits": "Forest. Dark.",
+    "clues": 1,
+    "code": "10619",
+    "double_sided": true,
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 20,
+    "faction_code": "mythos",
+    "illustrator": "Lin Hsiang",
+    "name": "Western Woods",
```
Comment (fspoettel) [inline pack/fhv/fhvc.json:466]:
```suggestion
    "name": "Corpse Grove",
```

**Review by fspoettel — CHANGES_REQUESTED** [review]

**Pull request merged**